spec-tests/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     implementation libraries["jacksonJsr310"]
     implementation libraries["assertj"]
     implementation libraries["commonsCodec"]
+    implementation libraries["guava"]
```

Comment:
What is this for?

Reply:
There were a lot of instances where we needed to get a single element out of a set and guava has a nice Iterables.getOnlyElement. Alternatively we could use a stream to get the element after we assert the set size is one, but the length is a little long.
`product.getBranding().stream().findAny().get()`

Reply:
ack